grpc/src/test/resources/test_jks/generate_jks.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
All files in `grpc/src/test/resources/test_jks/` directory are generated from this script.